src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -16,6 +16,8 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp2.0-Unix-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp2.0-Unix-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp2.1-Unix-Debug|AnyCPU'" />
```

Comment:
`netcoreapp` (versionless) is equivalent to netcoreapp2.1 so you can leave it as `netcoreapp`. I will go ahead and update your branch with the right confiurations as we need to keep netcoreapp2.0, netfx and netstandard as a `PackageConfiguration` and `netcoreapp` as a build configuration.

Reply:
Also the build configurations in the csproj are only for Visual Studio to understand the different configurations that this project can target, but the real configurations we use to build are in [configurations.props](https://github.com/dotnet/corefx/blob/master/src/System.Drawing.Common/src/Configurations.props) file

Reply:
Sounds good, thanks!

Reply:
I just pushed, you can take a look at what I did in commit: https://github.com/dotnet/corefx/pull/25134/commits/2aafcc4a7ad8c35d03532aa03c771c44c196dbd1